src/applications/travel-pay/components/TravelClaimCard.jsx

```diff
@@ -1,22 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { format } from 'date-fns';
-
-function formatDateTime(datetimeString) {
```

Comment:
Does this platform utility do what we're looking for? https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/utilities/date/index.js

Reply:
There's been at least [discussion](https://dsva.slack.com/archives/C04868KS69L/p1700242013495049) from platform teams to stop all new usages of Moment.js, which that utility uses, so I'd rather avoid it. The main reason I moved it to its own util file rather than just call `format` in place was to use it in a unit test that was having timezone issues in CI, and also figuring there's a chance down the line that it gets used in more places than just `TravelClaimCard`, e.g., a claim details page

Reply:
sold, thanks for the context.